pkg/csi/service/common/authmanager.go

```diff
@@ -326,16 +339,103 @@ func getDatastoresWithBlockVolumePrivs(ctx context.Context, vc *vsphere.VirtualC
 }
 
 // Creates a map of vsan datastores to file service status (enabled/disabled).
```

Comment:
this is not correct right as this only contains vsan datastores which have file service enabled. so there will be no disabled status at all.